### PR TITLE
Describe attack where AS can detect presence of specific measurements

### PR DESCRIPTION
**Pull request by ShivanKaul**

### CODE REVIEW

File: draft-dss-star.md [hunk 1]
```diff
@@ -881,6 +881,13 @@ they use, which reduces the window in which this attack can be launched ({{sec-r
 Note that such attacks can also be limited in scope by maintaining out-of-band protections
 against entities that attempt to launch large numbers of queries in short time periods.
 
+### Detecting Presence of Specific Measurements
+
+Related to {{dictionary-attacks}}, a malicious Aggregation Server could detect the presence of
+a specific measurement in the set of received encrypted reports by querying the Randomness Server 
+for that measurement. Similar to {{dictionary-attacks}}, this attack is also mitigated by regularly
+rotating the VOPRF key and out-of-band protections such as IP-based restrictions.
```
Comment (chris-wood) [inline draft-dss-star.md:889]: I'm not really sure I understand what this PR is trying to do. This _is_ a dictionary attack. Why is it being called out as something separate?

Comment (ShivanKaul) [reply draft-dss-star.md:889]: That was my initial reaction to Watson's email as well, but I think it's slightly different in the sense that the server doesn't necessarily need to launch multiple queries, even just one would be sufficient for detecting if the measurement you're looking for is in the set. So you're not trying out multiple words as in a dictionary attack. It's a small clarification, if we don't think it's worth mentioning I'm happy to close.

Comment (chris-wood) [reply draft-dss-star.md:889]: Yeah, I think we ought to close, or at least merge this discussion into the earlier section on dictionary attacks.

Comment (ShivanKaul) [reply draft-dss-star.md]: Merged discussion into Dictionary Attack section, looks better now IMO.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**